src/Tizen.Wearable.CircularUI.Forms.Renderer/CircleStepperRenderer.cs

```diff
@@ -55,6 +57,18 @@ protected override void OnElementChanged(ElementChangedEventArgs<CircleStepper>
 
                     SetNativeControl(spinner);
                     Control.ValueChanged += OnValueChanged;
+
+                    // This event will be occured on Tizen 4.0
+                    _genlistShow = new ElmSharp.SmartEvent(Control, "genlist,show");
```

Comment:
How about changing the event source dynamically according to the platform version by using system info?

Reply:
Fixed.